src/components/sound.js

```diff
@@ -66,7 +66,12 @@ module.exports.Component = registerComponent('sound', {
 
   remove: function () {
     this.el.removeObject3D('sound');
-    this.sound.disconnect();
+    try {
```

Comment:
Is there a way to know if the sound is connected or not instead of doing a try/catch?